DXRModules/DeusEx/Classes/DXRHalloween.uc

```diff
@@ -1,9 +1,5 @@
 class DXRHalloween extends DXRActorsBase;
```

Comment:
this class no longer needs to retain state, so we can make it transient which is safer for backwards compatibility with code changes and old saves
```suggestion
class DXRHalloween extends DXRActorsBase transient;
```